### PR TITLE
Test: add a contact from community's members list

### PR DESCRIPTION
**Pull request by Valentina1133**
Added test for adding contact from community channel

If you have any suggestions what else to add in this test - let me know

Screenshot 2024-05-22 at 18 38 51

CI run:
https://ci.status.im/job/status-desktop/job/e2e/job/manual/1930/

### CODE REVIEW

**Review by antdanchenko — APPROVED** [review]

File: squish.ini [hunk 1]
```diff
@@ -62,4 +62,6 @@ attachableAUT/AUT_150418 = "localhost:61500"
 attachableAUT/AUT_150532 = "localhost:61500"
 attachableAUT/AUT_150639 = "localhost:61500"
 attachableAUT/AUT_150925 = "localhost:61500"
+attachableAUT/AUT_171350 = "localhost:61500"
+attachableAUT/AUT_171701 = "localhost:61500"
```
Comment (anastasiyaig) [inline squish.ini:67]: I don't think we need to commit this file @Valentina1133

Comment (Valentina1133) [reply squish.ini:67]: Ooops! Will remove it, thank you 🙏🏻

Comment (Valentina1133) [reply squish.ini]: Done

File: tests/communities/test_communities_adding_contact_from_channel.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+from . import marks
+import configs
+import constants
+from constants import UserAccount
+
+pytestmark = marks
+
+
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/736170',
+                 'Add contact from community channel')
```
Comment (anastasiyaig) [inline tests/communities/test_communities_adding_contact_from_channel.py:20]:
```suggestion
                 'Add a contact from community's member list')
```

File: tests/communities/test_communities_adding_contact_from_channel.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+@pytest.mark.case(736170)
+def test_add_contact_from_community_channel_chat(multiple_instances):
```
Comment (anastasiyaig) [inline tests/communities/test_communities_adding_contact_from_channel.py:22]:
```suggestion
def test_add_a_contact_from_community_member_list(multiple_instances):
```

**Comment by Valentina1133** [comment]
> alright @Valentina1133 i finally got the idea what test is doing , suggest to rename the test itself :D

Yes, you are right, now it's more clear, thanks! I committed your suggestions

**Review by anastasiyaig — APPROVED** [review]

**Comment by anastasiyaig** [comment]
thank you !

**Pull request merged**